open_critter:
    # gazebo_ros_control:
    #   pid_gains:
    #     front_left_coxa_joint:
    #       p: 2.0
    #       i: 0.0
    #       d: 0.05
    #     front_right_coxa_joint:
    #       p: 2.0
    #       i: 0.0
    #       d: 0.05
    #     rear_right_coxa_joint:
    #       p: 2.0
    #       i: 0.0
    #       d: 0.05
    #     rear_left_coxa_joint:
    #       p: 2.0
    #       i: 0.0
    #       d: 0.05
    #     front_left_femur_joint:
    #       p: 2.0
    #       i: 0.0
    #       d: 0.05
    #     front_right_femur_joint:
    #       p: 2.0
    #       i: 0.0
    #       d: 0.05
    #     rear_right_femur_joint:
    #       p: 2.0
    #       i: 0.0
    #       d: 0.05
    #     rear_left_femur_joint:
    #       p: 2.0
    #       i: 0.0
    #       d: 0.05
    #     front_left_tibia_joint:
    #       p: 2.0
    #       i: 0.0
    #       d: 0.05
    #     front_right_tibia_joint:
    #       p: 2.0
    #       i: 0.0
    #       d: 0.05
    #     rear_right_tibia_joint:
    #       p: 2.0
    #       i: 0.0
    #       d: 0.05
    #     rear_left_tibia_joint:
    #       p: 2.0
    #       i: 0.0
    #       d: 0.05

    joint_state_controller:
        type: joint_state_controller/JointStateController
        publish_rate: 20

    front_left_coxa:
        type: position_controllers/JointPositionController
        joint: front_left_coxa_joint

    front_right_coxa:
        type: position_controllers/JointPositionController
        joint: front_right_coxa_joint

    rear_right_coxa:
        type: position_controllers/JointPositionController
        joint: rear_right_coxa_joint

    rear_left_coxa:
        type: position_controllers/JointPositionController
        joint: rear_left_coxa_joint

    front_left_femur:
        type: position_controllers/JointPositionController
        joint: front_left_femur_joint

    front_right_femur:
        type: position_controllers/JointPositionController
        joint: front_right_femur_joint

    rear_right_femur:
        type: position_controllers/JointPositionController
        joint: rear_right_femur_joint

    rear_left_femur:
        type: position_controllers/JointPositionController
        joint: rear_left_femur_joint

    front_left_tibia:
        type: position_controllers/JointPositionController
        joint: front_left_tibia_joint

    front_right_tibia:
        type: position_controllers/JointPositionController
        joint: front_right_tibia_joint

    rear_right_tibia:
        type: position_controllers/JointPositionController
        joint: rear_right_tibia_joint

    rear_left_tibia:
        type: position_controllers/JointPositionController
        joint: rear_left_tibia_joint
